src/Perlang.ConsoleApp/Program.cs

```diff
@@ -388,7 +416,9 @@ private void ParseAndPrint(string source)
 
         private void PrintBanner()
         {
-            standardOutputHandler($"Perlang Interactive REPL Console ({CommonConstants.GetFullVersion()}, built from git commit {CommonConstants.GitCommit})");
+            standardOutputHandler($"Perlang Interactive REPL Console (\x1B[1m{CommonConstants.GetFullVersion()}\x1B[0m, built from git commit {CommonConstants.GitCommit})");
+            standardOutputHandler("Type \x1B[36;1m/help\x1B[0m for more information or \x1B[36;1m/quit\x1B[0m to quit the program.");
+            standardOutputHandler(String.Empty);
```

Comment:
The careful reader will notice that this makes quite heavy use of ANSI colouring sequences. This should even work on Windows nowadays, as of Windows 10 version 1909 (released in November 2019). I haven't verified it, but will test it at some point after merging this.

The REPL is always expected to be executed in "interactive" mode, but we should probably try to auto-detect the support for colour at some point. Or at the very least, add a `--[no-]color` option to be able to turn it on or off.

(Side note, but I know I might seem completely inconsistent with the "color" vs "colour" usage above. but I'm not sure I really am. I think that `--color` makes more sense as a command line option, since more people are likely to expect the Americanized spelling of this word. At the same time, I think I learned the British spellings in school (in Sweden where I grew up), and I kind of like it better for some odd reason. It feels more colourful or something... :smirk:)

Reply:
Hmm... apparently not, after all (?). It doesn't look very pretty at all. We probably need to detect Windows and disable colors for now.

![image](https://user-images.githubusercontent.com/630613/154847868-28838f17-2943-4067-9007-cecda53bd0c4.png)